weasyl/config.py

```diff
@@ -1,14 +1,14 @@
 from __future__ import absolute_import
```

Comment:
A preliminary note: The ``__future__`` is now, and not needed on Python 3. (I don't think this _hurts_ anything if it remains until a cleanup PR, but it is an unnecessary relic from Python 2.7.)